libraries/chain/database.cpp

```diff
@@ -2616,6 +2620,13 @@ void database::_apply_block( const signed_block& next_block )
       FC_ASSERT( block_size <= gprops.maximum_block_size, "Block Size is too Big", ("next_block_num",next_block_num)("block_size", block_size)("max",gprops.maximum_block_size) );
    }
 
+   if( block_size < STEEMIT_MIN_BLOCK_SIZE )
```

Comment:
Is this even necessary? A block size less than `STEEMIT_MIN_BLOCK_SIZE` cannot exist because the fields are not optional. If we want this check, I think it should at the very least exist in `push_block`, not `_apply_block` that way it only happens while syncing. Even better, it should be inside `#ifdef STEEMIT_TESTNET` (or whatever the macro is) so that we catch it on CI (there are plenty of empty blocks generated during our tests) without incurring the cost in production.